iothub/device/src/ClientPropertyCollection.cs

```diff
@@ -48,17 +47,17 @@ public void AddComponentProperty(string componentName, string propertyName, obje
 
         /// <inheritdoc path="/remarks" cref="AddRootProperty(string, object)" />
         /// <inheritdoc path="/seealso" cref="AddInternal(IDictionary{string, object}, string, bool)" />
-        /// <inheritdoc path="/exception['ArgumentException']" cref="AddRootProperty(string, object)" />
         /// <summary>
         /// Adds the value to the collection.
         /// </summary>
         /// <param name="componentName">The component with the properties to add.</param>
         /// <param name="properties">A collection of properties to add.</param>
+        /// <exception cref="ArgumentException">A property name in <paramref name="properties"/> already exists in the collection.</exception>
```

Comment:
This API needs to inherit 1 of the 2 exceptions listed under `AddRootProperty`, so mentioning it explicitly